cdc/processor/sourcemanager/engine/engine.go

```diff
@@ -11,16 +11,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package sorter
+package engine
 
 import (
 	"github.com/pingcap/tiflow/cdc/model"
 )
 
-// EventSortEngine is a storage engine to store and sort CDC events.
-// Every changefeed will have one EventSortEngine instance.
+// SorterEngine is a storage engine to store and sort CDC events.
+// Every changefeed will have one SorterEngine instance.
 // NOTE: All interfaces are thread-safe.
-type EventSortEngine interface {
+type SorterEngine interface {
```

Comment:
Seems `SortEngine` is better than `SorterEngine`, just like `StorageEngine` or `RaftEngine`.

Reply:
Good idea! Thanks!